packages/utils/src/utils.ts

```diff
@@ -1,5 +1,6 @@
 import { TelescopeOptions, TelescopeOption, ProtoRef } from '@cosmology/types';
 import * as dotty from 'dotty';
+import os from 'os';
```

Comment:
not exactly sure if we should use this — one day could be we run this in the browser